L1Trigger/L1TTrackMatch/plugins/L1TrackerEtMissProducer.cc

```diff
@@ -13,158 +14,85 @@
 #include "DataFormats/L1TCorrelator/interface/TkEtMiss.h"
 #include "DataFormats/L1TCorrelator/interface/TkEtMissFwd.h"
 #include "DataFormats/L1TrackTrigger/interface/TTTypes.h"
-#include "DataFormats/L1Trigger/interface/Vertex.h"
 #include "DataFormats/Math/interface/LorentzVector.h"
 #include "FWCore/Framework/interface/Event.h"
 #include "FWCore/Framework/interface/EventSetup.h"
 #include "FWCore/Framework/interface/Frameworkfwd.h"
 #include "FWCore/Framework/interface/MakerMacros.h"
-#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/global/EDProducer.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 
-// detector geometry
-#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
-#include "Geometry/Records/interface/TrackerTopologyRcd.h"
-#include "MagneticField/Engine/interface/MagneticField.h"
-#include "MagneticField/Records/interface/IdealMagneticFieldRecord.h"
-
 using namespace l1t;
 
-class L1TrackerEtMissProducer : public edm::stream::EDProducer<> {
+class L1TrackerEtMissProducer : public edm::global::EDProducer<> {
 public:
   typedef TTTrack<Ref_Phase2TrackerDigi_> L1TTTrackType;
   typedef std::vector<L1TTTrackType> L1TTTrackCollectionType;
-
-  typedef Vertex L1VertexType;
-  typedef VertexCollection L1VertexCollectionType;
+  typedef edm::RefVector<L1TTTrackCollectionType> L1TTTrackRefCollectionType;
 
   explicit L1TrackerEtMissProducer(const edm::ParameterSet&);
   ~L1TrackerEtMissProducer() override;
```

Comment:
```suggestion
  ~L1TrackerEtMissProducer() override = default;
```
(and remove the implementation at [L60](https://github.com/cms-sw/cmssw/pull/38442/files#diff-39e08c7047f33653271c15f8ccfb123cf5084fe998aaa84a7028c2593a1114c4R60-R61)